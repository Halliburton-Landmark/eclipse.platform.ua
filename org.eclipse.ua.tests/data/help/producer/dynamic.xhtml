<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<!--
   This file is used to test whether or not the dynamic producer will be used
   when loading an xhtml document from outside a doc.zip. The paragraph at the
   bottom should always be filtered out by the producer.
-->

<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>dynamic.xhtml</title>
</head>
<body>

<h2>dynamic.xhtml</h2>
<p>
This page contains filtered content.
</p>
<p filter="os=invalid">
This paragraph, which has filter attribute "os=invalid" should always be filtered out.
</p>
<p filter="os!=invalid">
This paragraph, which has filter attribute "os!=invalid" should never be filtered out.
</p>
<p filter="plugin=my.invalid.plugin.id.012345">
This paragraph, which has filter attribute "plugin=my.invalid.plugin.id.012345" should always be filtered out.
</p>
<p filter="plugin!=my.invalid.plugin.id.012345">
This paragraph, which has filter attribute "plugin!=my.invalid.plugin.id.012345" should never be filtered out.
</p>
<p>
   <filter name="plugin" value="my.invalid.plugin.id.012345"/>
   This paragraph, which has filter element name="plugin", value="my.invalid.plugin.id.012345",
   should always be filtered out.
</p>
<p>
   <filter name="plugin" value="!my.invalid.plugin.id.012345"/>
   This paragraph, which has filter element name="plugin", value="!my.invalid.plugin.id.012345",
   should never be filtered out.
</p>

<include path="org.eclipse.ua.tests/data/help/producer/include.xhtml/my.paragraph.id"/>
<include path="org.eclipse.ua.tests/data/help/producer/dynamicInZip.xhtml/paragraphInZip"/>

</body>
</html>
