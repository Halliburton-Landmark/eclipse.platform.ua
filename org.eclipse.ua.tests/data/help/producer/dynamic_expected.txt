<!--
   This file is used to test whether or not the dynamic producer will be used
   when loading an xhtml document from outside a doc.zip. The paragraph at the
   bottom should always be filtered out by the producer.
--><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>

<title>dynamic.xhtml</title>
</head>
<body>

<h2>dynamic.xhtml</h2>
<p>
This page contains filtered content.
</p>

<p filter="os!=invalid">
This paragraph, which has filter attribute "os!=invalid" should never be filtered out.
</p>

<p filter="plugin!=my.invalid.plugin.id.012345">
This paragraph, which has filter attribute "plugin!=my.invalid.plugin.id.012345" should never be filtered out.
</p>

<p>
   <filter name="plugin" value="!my.invalid.plugin.id.012345"/>
   This paragraph, which has filter element name="plugin", value="!my.invalid.plugin.id.012345",
   should never be filtered out.
</p>

<p id="my.paragraph.id">
   This paragraph comes from <code>include.xhtml</code>.
</p>
<p id="paragraphInZip">
This paragraph comes from dynamicInZip.xhtml in doc.zip.
</p>

</body>
</html>